### ISSUE TITLE
Empty light grey rectangle as picture for blog posts

### ISSUE THREAD

**Issue by StefanSalewski** [h-enk/doks#1040]
Dear Sir,

after working since a few days with Hugo and your doks theme, most seems to work fine. But for blogs, I had from the beginning the issue, that the posts look very strange. I think, I have now found out, that a large empty light grey rectangle is put in front of the posts always. From  showcase https://www.divecoaching.com.au/blog/ it seems to be possible to provide own pictures, but I have no idea how exactly that may be done. Have spent some time with google search, and found some reports from people with similar problems, but no solution. Adding PNG pictures to the text area of blog posts works. But using images: ["mypic.png"] in index.md frontmatter seems to have no effect. Picture is in same directory as index.md. Maybe I am doing it fully wrong, I have found no concrete description how to do it. Disabling pictures, or reducing size seems to not possible in v0.5, as others wrote? Luckely blogs are not important for my page, but I have spent a few hours trying to get it working now. Well, the rest seems to work fine with local server, I assume and hope that deploying will work as well. Otherwise, a few days of work would be wasted :-)

I hope your v1.0 will arrive soon.

Best regards,

Stefan Salewski

**Comment by StefanSalewski** [comment]
I also tried absolute paths, but that makes no difference. Github has a link under Demo to https://doks.netlify.app/, and when I click there on Blog I get the same light grey rectangle.

**Comment by StefanSalewski** [comment]
And finally, after four hours:

```
images: ["yellow-rose.jpg"]
```

So for the frontmatter of index.md of blog post, only jpg files work, while in the text content, png files work fine. Still, an option to shrink the image or do not use it at all would be fine. I think all my googlings gave one hint how one may patch some files so that images are not shown at all, I will see if I can find it again. But working jpg files is good enough for me, now I know that I did it not just wrong. Maybe you have seen that I also added some text to the discussions, e.g. about syntax highlight.
